packages/docs/locales/en.json

```diff
@@ -1667,7 +1667,8 @@
     "i18n": "I18n",
```

Comment:
Some translations look to be broken:

<img width="988" alt="image" src="https://user-images.githubusercontent.com/5394573/222278534-68f02210-35f7-4e2e-bfc8-9c6f1f30bedd.png">